src/BuildLoop.cpp

```diff
@@ -217,11 +219,16 @@ static HashDigest ComputeLeafInputSignature(BuildQueue* queue, ThreadState* thre
 
     if (sigoffline == NULL)
         CroakErrno("Failed opening offline signature ingredients for reading.");
-    
+
+#if TUNDRA_WIN32
+    const char* username = getenv("USERNAME");
+    if (username != nullptr)
+        fprintf(sig, "Signature generated by %s\n", username);
+#else
```

Comment:
Uhm, isn't this something completely different? I guess USERNAME is the name of your user account vs hostname, which is the name of the machine? Now, it may make sense to have both, or only one, but not something completely different on windows and mac/linux. (FYI, `getenv("USER")` works on mac).